### ISSUE TITLE
Neutralino.window.create('/') and Events

### ISSUE THREAD

**Issue by Kantyash** [neutralinojs/neutralinojs#864]
**Expected behavior**
I click a button and create a 2nd instance of the app with _window.create('/');_. When I broadcast an event both instances receive it.

**Actual behavior**
When I broadcast an event, no matter if I use _events.broadcast_, _events.dispatch_ or _app.broadcast_, the event only registers in the current window.

I'm new to this so maybe I don't understand how this works but I thought this is how it's supposed to work? If I misunderstood, please correct me and let me know if there is a better way of sending an event to all app instances. I'm using _PIXI.js_ in the project but I don't think it's relevant in this scenario. The way I did it before was with .storage but that's really annoying.

### Here's my code:

**We make an event listener**
```
async function eventing(){
    await Neutralino.events.on('balrog', shakira);
};
eventing();
```
**Then we make a function to call the event (how we call this function is irrelevant I think):**
```
async function goevent(){
    await Neutralino.events.broadcast('balrog');
};
```
**And here's the stupid function that gets called by the event:**
```
function shakira(){
    console.log('shakira');
    let movingoX =  Math.random() * (window.screen.availWidth / 2 - 1) + 1;
        let movingoY =  Math.random() * (window.screen.availHeight / 2 - 1) + 1;
        window.Neutralino.window.move(movingoX, movingoY);
        let sizeoY =  Math.random() * (window.screen.availHeight / 1.5 - 288) + 288;
        let sizeoX =  (sizeoY / 3) * 4; 
        window.Neutralino.window.setSize({
            width: sizeoX,
            height: sizeoY
        });
};
```
**And the code I use for making a 2nd window, if it matters:**
`window.Neutralino.window.create('/');`

Basically, I'd expect both windows to move and resize on click (when the event gets broadcasted) but only the window I clicked the button in does.

**Running on:**
NeutralinoJs Version: 3.3.0
NeutralinoJs CLI version: 4.4.0
Windows 10

**Comment by Alamantus** [comment]
I'm having this same problem on MacOS using NeutralinoJs 10.0.0 (though I had to build from source to get the x64 version working properly if that makes a difference)

Event broadcasting between windows seems to be working in Windows for me though.

**Comment by mathiasayivor** [comment]
I'm experiencing the same issue on Windows 10
CLI: v11.5.0
Neutralinojs binaries: v6.2.0
Neutralinojs client: v6.2.0

**Comment by taqi110913** [comment]
I'm also experiencing the same issue.
Device: Windows 11
neu CLI: v11.7.2
Neutralinojs binaries: v6.9.0
Neutralinojs client: v6.9.0
